test_data/config.toml

```diff
@@ -12,6 +12,13 @@ post_cmd = '''
     mv empty.json "$main_output_dir/empty.json"
 '''
 
+[resolve.failed-resolution]
```

Comment:
There's already a convention of using underscores in this file, can we stick to that?

Reply:
sure, tunnel vision :)